platform/app/prisma/migrations/20260818090000_grant_resource_facts/migration.sql

```
@@ -0,0 +1,23 @@
+    "organizationId" TEXT NOT NULL,
+    "projectId" TEXT NOT NULL,
+    "viewCount" INTEGER NOT NULL DEFAULT 0,
+    "lastViewedAt" TIMESTAMP(3),
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "GrantUsage_pkey" PRIMARY KEY ("grantId")
+);
+
+-- CreateIndex
+CREATE INDEX "Grant_projectId_resourceKind_scopeId_idx" ON "Grant"("projectId", "resourceKind", "scopeId");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)